packages/platform-sdk-ark/src/services/ledger.ts

```diff
@@ -41,4 +58,42 @@ export class LedgerService implements Contracts.LedgerService {
 	public async signMessage(path: string, payload: Buffer): Promise<string> {
 		return this.#transport.signMessageWithSchnorr(path, payload);
 	}
+
+	public async scan(): Promise<Contracts.WalletData[]> {
+		const pageSize = 5;
```

Comment:
Is chunk size 5 fine? We'll later make wallet requests 5 at a time. Depending on how many at a time the api supports we could have bigger pages.

Reply:
With ARK we can send batches of up to 50 addresses in a single request if ever needed.